### ISSUE TITLE
Which dataset is used for validation during pretraining?

### ISSUE THREAD

**Issue by ducnx** [hhi-aml/ecg-selfsupervised#2]
In function [val_dataloader](https://github.com/hhi-aml/ecg-selfsupervised/blob/f3e93ea297a027a7dd0472c3c379445d1d191cbc/ecg_datamodule.py#L71), you use the `eval_dataset` config to generate validation set during pretraining. In `bolts_config.yaml`, under `eval_dataset` config, the path to PTB-XL dataset is set as the target folder. Why don't you use a fold from `All` datasets as the validation set?
Please clarify if I miss any detail. Thanks in advance.
